index.js

```diff
@@ -224,3 +224,16 @@ function popupProject(y) {
     body.style.overflowY = 'scroll';
   }
 }
+
+const cform = document.querySelector('.contact-form');
+const errMessage = document.querySelector('.validate');
+const mail = document.querySelector('.contactme');
+
+cform.addEventListener('submit', (event) => {
+  event.preventDefault();
+  if (mail.value !== mail.value.toLowerCase()) {
+    errMessage.style.display = 'block';
+  } else {
```

Comment:
- [ ] You did an amazing job adding the error message but kindly remove it near the submit button as it was part of the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md). Keep shining 💪 

> - If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.

>